Content.Client/Construction/ConstructionButton.cs

```diff
@@ -27,12 +27,24 @@ protected override void Initialize()
         {
             base.Initialize();
 
+            AnchorLeft = 1.0f;
```

Comment:
You can do `SetAnchorPreset(LayoutPreset.BottomRight)` to set all these anchors.

Reply:
Done